lib/serverengine/socket_manager.rb

```diff
@@ -68,7 +68,8 @@ def self.generate_path
             end
           end
         else
-          '/tmp/SERVERENGINE_SOCKETMANAGER_' + Time.now.to_s.gsub(' ', '') + '_' + Process.pid.to_s
+          base_dir = (ENV['SERVERENGINE_SOCKETMANAGER_SOCK_DIR'] || '/tmp')
+          File.join(base_dir, 'SERVERENGINE_SOCKETMANAGER_' + Time.now.to_s.gsub(' ', '') + '_' + Process.pid.to_s)
```

Comment:
How about to switch to use `Time.now.utc.iso8601`?
It creates timezone-independent path and doesn't contain whitespaces, like: `2017-03-01T16:14:12+09:00`.